src/interfaces/index.ts

```diff
@@ -74,3 +74,27 @@ export interface State {
     isOpen: boolean;
   };
 }
+
+export interface IChartProps {
```

Comment:
Ok, its good, but in my next push I will added a type, I don´t know if you nedeed that, you can use my type and delete your interface.